packages/hooks/src/hooks/useWallet.ts

```diff
@@ -22,16 +22,56 @@ export function useWallet() {
     connected,
     provider,
     correctNetwork,
+    network,
   } = context;
 
   React.useEffect(() => {
     if (userAddress && provider && chainId === NETWORKS.mainnet) {
-      provider.lookupAddress(userAddress).then((address) => {
+      provider.lookupAddress(userAddress).then(address => {
         setEns(address as string);
       });
     }
   }, [userAddress, provider]);
 
+  const switchToCorrectNetwork = async () => {
+    if (window.ethereum) {
+      try {
+        console.log('chainId', { chainId });
+        console.log('requiredNetwork', { network });
+        // check if the chain to connect to is installed
+        await window.ethereum.request({
+          method: 'wallet_switchEthereumChain',
+          params: [{ chainId: `0x${network}` }], // chainId must be in hexadecimal numbers
+        });
+      } catch (error) {
+        console.error(error);
+        // This error code indicates that the chain has not been added to MetaMask.
+        // If it is not, then install it into the user MetaMask
+        // @ts-ignore
+        if (error.code === 4902) {
+          try {
+            await window.ethereum.request({
+              method: 'wallet_addEthereumChain',
+              params: [
+                {
+                  chainId: '0x61',
+                  rpcUrl: 'https://data-seed-prebsc-1-s1.binance.org:8545/',
+                },
+              ],
+            });
+          } catch (addError) {
+            console.error(addError);
+          }
+        }
+      }
+    } else {
+      // if no window.ethereum then MetaMask is not installed
+      alert(
+        'MetaMask is not installed. Please consider installing it: https://metamask.io/download.html'
```

Comment:
```suggestion
        'Switching networks automatically is only supported in MetaMask. https://metamask.io/download.html'
```

Can you change the error message?

Reply:
Sure, I'll update it.